group_vars/location_rhxb/networks.yml

```diff
@@ -64,35 +64,41 @@ networks:
     ipv6_subprefix: -7
     ptp: true
 
+  - vid: 17
+    role: mesh
+    name: mesh_dragon
+    prefix: 10.31.126.135/32
```

Comment:
This IP is already used for mesh_no_5ghz

Reply:
Thanks.  Fixed